components-java-sdk/src/test/java/io/dapr/components/wrappers/OutputBindingGrpcComponentWrapperTest.java

```diff
@@ -81,9 +97,42 @@ List<DynamicTest> ping() {
 
   @Test
   void invoke() {
+    final InvokeResponse mockedInvokeResponse = new InvokeResponse(
+        ByteString.copyFromUtf8(INVOKE_RESPONSE_DATA),
+        RESPONSE_METADATA,
+        CONTENT_TYPE);
+
+    when(mockComponent.invoke(any())).thenReturn(Mono.just(mockedInvokeResponse));
+
+    final Bindings.InvokeResponse response = client.invoke(Bindings.InvokeRequest.newBuilder()
+            .setData(ByteString.copyFromUtf8(INVOKE_REQUEST_DATA))
+            .setOperation(OPERATION_NAME)
+            .putAllMetadata(REQUEST_METADATA)
+        .build());
+    var capturedRequest = ArgumentCaptor.forClass(InvokeRequest.class);
+
+    verify(mockComponent, times(1)).invoke(capturedRequest.capture());
+    // Validate how request was translated
+    final InvokeRequest translatedRequest = capturedRequest.getValue();
+    assertThat(translatedRequest.data().toStringUtf8()).isEqualTo(INVOKE_REQUEST_DATA);
+    assertThat(translatedRequest.metadata()).isEqualTo(REQUEST_METADATA);
+    assertThat(translatedRequest.operation()).isEqualTo(OPERATION_NAME);
+    // Validate response
+    assertThat(response).isNotNull();
+    assertThat(response.getData().toStringUtf8()).isEqualTo(INVOKE_RESPONSE_DATA);
+    assertThat(response.getMetadataMap()).isEqualTo(RESPONSE_METADATA);
+    assertThat(response.getContentType()).isEqualTo(CONTENT_TYPE);
   }
 
   @Test
   void listOperations() {
+    when(mockComponent.listOperations()).thenReturn(Mono.just(List.of(OPERATION_NAME)));
+
+    final Bindings.ListOperationsResponse response =
+        client.listOperations(Bindings.ListOperationsRequest.newBuilder().build());
+
+    verify(mockComponent, times(1)).listOperations();
+    assertThat(response).isNotNull();
+    assertThat(response.getOperationsList()).contains(OPERATION_NAME);
```

Comment:
Nit: this verifies that the returned list of operations containers the mock one, but not that it *doesn't* potentially contain others?

Reply:
Let me move that to equality. Fixed on 0d973a45064db67dd828225ec6e4a381db43e9de